### DISCUSSION TITLE
Rewrite of the project

### DISCUSSION THREAD

**Discussion by FelixKratz** [cmacrae/spacebar#86] [General]
Hey,
when I found this project I was totally stoked about a custom macOS menubar, but soon felt that the current implementation
is not really powerful. I have decided to rewrite the entire project with powerful new features and added flexibility. You guys might enjoy it: [https://github.com/FelixKratz/SketchyBar](https://github.com/FelixKratz/SketchyBar).

Cheers!

**Comment by cmacrae** [comment]
Oh wow, great changes & additions! 👏  
It would've been nice to receive these changes as contributions to spacebar and collaborate, but I understand wanting to pursue your own endeavors :)

I'm in the middle of [a rewrite of the project myself](https://github.com/cmacrae/spacebar/discussions/73), so I'll certainly be leaning on the great examples you've set out in SketchyBar!

**Comment by FelixKratz** [comment]
I only chose to open a new repo because I didn't want to nuke your code with my radical changes, I think of the original bar logic not much is left. 

How do you like the idea of us both working on SketchyBar as a "beta rewrite" channel and then merging the changes to the spacebar repo once we are happy with it?

**Comment by cmacrae** [comment]
> I only chose to open a new repo because I didn't want to nuke your code with my radical changes, I think of the original bar logic not much is left.

Sure, I can appreciate that

> How do you like the idea of us both working on SketchyBar as a "beta rewrite" channel and then merging the changes to the spacebar repo once we are happy with it?

I think that sounds like a great idea! You've already achieved a lot and there's much I can learn from you -- I may have some additions to contribute from the re-write I was already working on, too :)

**Comment by cmacrae** [comment]
@FelixKratz What's the best way for us to chat? :) Are you on Discord? Or would you prefer e-mail?

**Comment by FelixKratz** [comment]


**Reply by cmacrae** [reply]
Dropped you a friend request :)

**Comment by andynameistaken** [comment]
Hi, are you still rewriting this project?
